astronomer/providers/amazon/aws/example_dags/example_redshift_cluster_management.py

```diff
@@ -58,6 +58,7 @@
             {"Key": "Purpose", "Value": "ProviderTest"},
         ],
         wait_for_completion=True,
+        max_attempt=30,
```

Comment:
is there any other way of waiting for the cluster (using boto3 waiters concept) than harcoding it?

Reply:
https://github.com/apache/airflow/blob/main/airflow/providers/amazon/aws/operators/redshift_cluster.py#L328-L334
 the OSS library uses waiters and we are sending value to the operator.